AzureTable/Microsoft.DataTransfer.AzureTable/RemoteLogging/IRemoteLogging.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.DataTransfer.AzureTable
+{
+    /// <summary>
+    /// The interface for remotely logging to Cosmos tables, when it is used as a sink endpoint.
+    /// This is beneficial to consolidate failure logs when multiple instances of data migration
+    /// tool are run simultaneously. 
+    /// </summary>
+    public interface IRemoteLogging
+    {
+        /// <summary>
+        /// Log failures that occurred during a data migration.
+        /// </summary>
+        /// <param name="partitionKey"></param>
```

Comment:
xmldoc is missing for all params here and below

Reply:
N/A here .. applied feedback to the interface in new location Core project.